### ISSUE TITLE
When unfilling outlet: IntegrityError: (IntegrityError) ORA-01400: cannot insert NULL into ("INV"."OUT"."ENTRY_DT")

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2350]
_Imported from trac ticket 2350.
Created by **sjohnson**
Opened in trac: May 28, 2013, 5:00:26 PM
Last modified in trac: Jun 4, 2013, 11:33:45 AM_
***
_by sjohnson:_
```
 IntegrityError: (IntegrityError) ORA-01400: cannot insert NULL into ("INV"."OUT"."ENTRY_DT")
[Tue May 28 16:17:27 2013] [error] [client 208.104.106.106]  'INSERT INTO outletinventory (outletid, productid, siteid, locationid, statusid, qty, cost, outletdate, spiff, costdate, receiptdate, commissiontype, "comment", serialnumber) VALUES (:outletid, :productid, :siteid, :locationid, :statusid, :qty, :cost, :outletdate, :spiff, :costdate, :receiptdate, :commissiontype, :"comment", :serialnumber)' {'"comment"': None, 'serialnumber': None, 'outletid': '0430336904', 'receiptdate': datetime.date(2012, 9, 27), 'outletdate': None, 'qty': Decimal('1'), 'statusid': 'OUT', 'cost': Decimal('175'), 'siteid': '13', 'locationid': 'OUTLET', 'costdate': datetime.date(2012, 9, 27), 'spiff': Decimal('0'), 'commissiontype': 'DCC', 'productid': '185123800'}
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Jun 4, 2013, 11:33:45 AM:_

(Changeset [[4209]](../commit/ce86b1794968c48ed07e925f648e60219bea515c))
* fixes #2350, Allow unfill/void of outlet inventory when orderline does not have outletdate (ENTRY_DT) for some reason. Fixes (IntegrityError) ORA-01400: cannot insert NULL into ("INV"."OUT"."ENTRY_DT") #isf, #mor

**Comment by kentbower** [comment]
_**kbower** trac status change on Jun 4, 2013, 11:33:45 AM:_ **closed**
